### ISSUE TITLE
On Edge: when opening dropdown [dropup]="true"  at the bottom of the page, the height of the page increases

### ISSUE THREAD

**Issue by brachagrinfeld** [valor-software/ng2-bootstrap#5331]
### Bug description or feature request:

On Edge: When opening dropdown [dropup]="true"  at the bottom of the page, the height of the page increases, and a space is added below the dropdown

`<div class="btn-group" dropdown [dropup]="isDropup" container="body" style="margin-top: 100%">`

### Plunker/StackBlitz that reproduces the issue:

https://stackblitz.com/edit/angular-rhd815?embed=1&file=app/dropup.html
The dropdown is located at the bottom of the page. if you cannot see it, please scroll down

### Versions of ngx-bootstrap, Angular, and Bootstrap:

ngx-bootstrap: 3.2.0

Angular: 7.2.2

Bootstrap: 4.2.1

### Solution suggestion

ngx-bootstrap is adding inline-style to the dropdown `ul` element

`<ul aria-labelledby="button-dropup" class="dropdown-menu show" style="top: auto; transform: translateY(-101%);">`

the style has to be:

`style="top: auto; transform: translateY(-1%); bottom: 100%"`

**Comment by henrysachs** [comment]
I would like to help on this.

**Comment by henrysachs** [comment]
Hi, I have a hard time reproducing this. When i open the Dropdown with your provided example the document height doesn't increase. I still built the lib locally and have copied your code and logged the document scrollheight. 

If I am doing something wrong here help me out. 

![Bild 01 10 19 um 20 30](https://user-images.githubusercontent.com/17173951/65989674-6903fe80-e48a-11e9-87f6-e9f4e5929fc3.jpg)

**Comment by nik72619c** [comment]
shall I finish this PR ?was working on this

**Comment by daniloff200** [comment]
yeah, if you already started working on it, @nik72619c , I can assign this to you

**Comment by nik72619c** [comment]
sure thing !

**Comment by daniloff200** [comment]
hey, @nik72619c 
what's your progress with that ?
